src/components/PoolOverview.tsx

```diff
@@ -91,6 +99,24 @@ function PoolOverview({
             <span className="label">{t("currencyReserves")}</span>
             <span>{`$${formattedData.reserve}`}</span>
           </div>
+          {formattedData.utilization && (
+            <div className="volume">
+              <span className="label">Utilization</span>
```

Comment:
pls add these titles to our en.json and zh.json translation files. (zh entry can be null)